plugins/qlcplus_4.12.1/presets.js

```diff
@@ -0,0 +1,1002 @@
+    isApplicable: cap => cap.type === `PanContinuous` && exportHelpers.isDecreasingSpeed(cap),
+    importCapability: () => ({
+      type: `PanContinuous`,
+      speedStart: `fast`,
+      speedEnd: `slow`
+    })
+  },
+  SpeedTiltSlowFast: {
+    isApplicable: cap => cap.type === `TiltContinuous` && exportHelpers.isIncreasingSpeed(cap),
+    importCapability: () => ({
+      type: `TiltContinuous`,
+      speedStart: `slow`,
+      speedEnd: `fast`
+    })
+  },
+  SpeedTiltFastSlow: {
+    isApplicable: cap => cap.type === `TiltContinuous` && exportHelpers.isDecreasingSpeed(cap),
+    importCapability: () => ({
+      type: `TiltContinuous`,
+      speedStart: `fast`,
+      speedEnd: `slow`
+    })
+  },
+  SpeedPanTiltSlowFast: {
+    isApplicable: cap => cap.type === `PanTiltSpeed` && (exportHelpers.isIncreasingSpeed(cap) || exportHelpers.isDecreasingDuration(cap)),
+    importCapability: () => ({
+      type: `PanTiltSpeed`,
+      speedStart: `slow`,
+      speedEnd: `fast`
+    })
+  },
+  SpeedPanTiltFastSlow: {
+    isApplicable: cap => cap.type === `PanTiltSpeed` && (exportHelpers.isDecreasingSpeed(cap) || exportHelpers.isIncreasingDuration(cap)),
+    importCapability: () => ({
+      type: `PanTiltSpeed`,
+      speedStart: `fast`,
+      speedEnd: `slow`
+    })
+  },
+
+  ColorMacro: {
+    isApplicable: cap => cap.type === `ColorPreset` || (cap.type === `WheelSlot` && cap.isSlotType(`Color`)),
+    importCapability: () => ({
+      type: `ColorPreset`,
+      helpWanted: `Which color can be selected at which DMX values?`
+    })
+  },
+  ColorWheel: {
+    isApplicable: cap => cap.type === `WheelRotation` && cap.wheels[0].type === `Color`,
+    importCapability: () => ({
+      type: `WheelRotation`,
+      wheel: ``,
+      speedStart: `slow`,
+      speedEnd: `fast`
+    })
+  },
+  ColorRGBMixer: { // basically this is also Hue
+    isApplicable: cap => false,
+    importCapability: () => ({
+      type: `Generic`
+    })
+  },
+  ColorCTOMixer: {
+    isApplicable: cap => cap.type === `ColorTemperature` && cap.colorTemperature[0].number === 0 && cap.colorTemperature[1].number < 0,
+    importCapability: () => ({
+      type: `ColorTemperature`,
+      colorTemperatureStart: `default`,
+      colorTemperatureEnd: `warm`,
+      helpWanted: `Can you provide exact color temperature values in Kelvin?`
+    })
+  },
+  ColorCTBMixer: {
+    isApplicable: cap => cap.type === `ColorTemperature` && cap.colorTemperature[0].number === 0 && cap.colorTemperature[1].number > 0,
+    importCapability: () => ({
+      type: `ColorTemperature`,
+      colorTemperatureStart: `default`,
+      colorTemperatureEnd: `cold`,
+      helpWanted: `Can you provide exact color temperature values in Kelvin?`
+    })
+  },
+  ColorCTCMixer: {
+    isApplicable: cap => cap.type === `ColorTemperature`,
+    importCapability: () => ({
+      type: `ColorTemperature`,
+      colorTemperatureStart: `cold`,
+      colorTemperatureEnd: `warm`,
```

Comment:
Most fixtures have low Kelvin to high Kelvin, so this should be warm…cold per default